docs/STUDY_PUBLISHING.md

```diff
@@ -1,17 +1,7 @@
-# Study Extension Publishing HOWTO
+# Study Extension Publishing
 
-## Getting Started
-
-Create account on https://addons.mozilla.org, navigate to the "Developer Hub", and choose "Submit a New Add-on".
-
-## Build your extension
-
-Starting with a github repository containing your study extension (such as [the Rally Study Template](https://github.com/mozilla-rally/study-template)):
-
-* run `web-ext build` to generate your extension in `web-ext-artifacts/`
-* upload the resulting `.zip` file in `web-ext-artifacts/` to the https://addons.mozilla.org Developer Hub
-
-It may take some time for your extension to be reviewed and available for download.
+Study extensions are unprivileged and hosted on addons.mozilla.org, and should follow the same experience as a normal third-party add-on.
```

Comment:
This should probably have a link for what `a normal third-party add-on` is, since honestly most people out in the world have no idea what that means 😆 

We might want to remove "third-party" as well, since the only people for whom a 1st party add-on is normal is Mozilla hehe.

Reply:
I just mentioned this at top-level, but this document is only for us not for partners. I'm not opposed to making it more accessible to folks that are interested though!